### ISSUE TITLE
Update Screen With Blank Text

### ISSUE THREAD

**Issue by Mykl1** [InFlames03/Oreo-Theme#10]
![screenshot_20180305-211215](https://user-images.githubusercontent.com/29133471/37012368-e7e08598-20b9-11e8-9ae7-0a7e8efdb672.png)
I am on a pixel gen 1 (Latest February Security Patch and Android 8.1) with the theme version eight (latest). There should be text telling me information about updates but there is none. 

If helpful - Substratum version -  953
- Andromeda version -  19

**Comment by InFlames03** [comment]
Yes I noticed that issue as well, but was slightly too late. I'll push a new update next weekend (fastest I can because of exams this week).

Thanks for the report!

**Comment by Mykl1** [comment]
@InFlames03 No problem, glad it's getting fixed 😀

**Comment by Mykl1** [comment]
@InFlames03 Don't mean to bug you but we're you able to put out the fix for the update screen, I don't think I ever saw one in the play store.

**Comment by InFlames03** [comment]
The fix has been done for a week, but I want to iron out more edges before I push it to the Play Store

**Comment by Mykl1** [comment]
@InFlames03 Ok, thanks for responding
